lua/utils/security.lua

```
@@ -0,0 +1,207 @@
+end
+
+-- Execute command safely
+-- @param cmd Command to execute
+-- @param force Skip confirmation for non-allowed commands
+-- @return string|nil Output or nil on error
+-- @return boolean|string Success flag or error message
+function M.safe_execute(cmd, force)
+  -- Check if command starts with an allowed prefix
+  local command_base = cmd:match("^%s*(%S+)")
+  local allowed = false
+  
```

Luacheck (1.2.0)

[warning] line contains only whitespace

(W611)